server/blob/blob-export-file/src/main/java/org/apache/james/blob/export/file/LocalFileBlobExportMechanism.java

```diff
@@ -46,7 +46,7 @@
 
     public static class Configuration {
 
-        private static final String DEFAULT_DIRECTORY_LOCATION = "file://var/blobExporting";
+        private static final String DEFAULT_DIRECTORY_LOCATION = "file://127.0.0.1/var/blobExporting";
```

Comment:
I think you mean file:///var/blobExporting which is the classic way to specify a local file, no?
But here I guess var is not /var but the var of James, so file://var could have a special meaning.

Reply:
I've just digged to the FileSystemImpl in James and realize that that is the var directory specified in James. It's alright to James, but when I open the exporting mail. I can see a part of the content:
```
file is located in file://var/blobExport/.....
```
Which is not friendly to the user perspective. Because I carries james knowledge.

Reply:
oh right, in the explanation text we must replace this by the real path of the file